quarkus-build-caching-extension/src/it/postbuild.groovy

```diff
@@ -0,0 +1,38 @@
+// Utility functions
+String getContent(String fileName) {
+    File f = new File(basedir, fileName)
+    assert f.exists()
+    return f.text
+}
+
+void checkBuildCacheDisabled(String logFile) {
```

Comment:
There is some strange spacing in the `contains()` invocations of these 3 fuctions.